internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go

```diff
@@ -398,7 +398,7 @@ func (s schedulerStorage) RunWithTransaction(ctx context.Context, transactionFun
 	s.transactionsMap[transactionID] = transaction
 
 	defer func() {
-		if err := recover(); err != nil {
+		if err := recover(); err != any(nil) {
```

Comment:
Why this?

Reply:
The linter in GOLAND was complaining about using nil with an `any` type. So I search for how to fix it.